mlflow/tracking/_model_registry/fluent.py

```diff
@@ -126,6 +127,19 @@ def _register_model(
         f"Created version '{create_version_response.version}' of model "
         f"'{create_version_response.name}'."
     )
+
+    if model_id := _parse_model_id_if_present(model_uri):
+        client.set_logged_model_tags(
+            model_id,
+            {
+                mlflow_tags.MLFLOW_REGISTERED_MODEL: json.dumps(
+                    {
+                        "name": create_version_response.name,
+                        "version": create_version_response.version,
+                    }
+                )
+            },
+        )
```

Comment:
@harupy If there's already a tag present for this logged model, can we append a new entry to the list? There may be multiple model versions registered from the same logged model